bin/oneoff/remove_authorized_teacher_permissions_from_students

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env ruby
+
+require_relative '../../dashboard/config/environment'
+
+DRY_RUN = true
+
+authorized_teacher_student_permissions = UserPermission.where(permission: UserPermission::AUTHORIZED_TEACHER).left_joins(:user).where('users.user_type': 'student')
+
+removed_permissions = 0
+
+authorized_teacher_student_permissions.each do |permission|
+  user = permission.user
+  next unless user.student?
+  next unless permission.permission = UserPermission::AUTHORIZED_TEACHER
```

Comment:
Yeah, I was being a little over cautious to avoid removing other permissions. Though I guess in this case, it's not a huge deal because students shouldn't actually have other permissions